src/explorer.ts

```diff
@@ -61,7 +61,13 @@ export class KubernetesExplorer implements vscode.TreeDataProvider<KubernetesObj
 
     private async getClusters(): Promise<KubernetesObject[]> {
         const clusters = await kubectlUtils.getClusters(this.kubectl);
-        return clusters.map((cluster) => new KubernetesCluster(cluster.name, cluster));
+        return clusters.map((cluster): KubernetesCluster => {
+            // TODO: this is slightly hacky...
+            if (cluster.name == 'minikube') {
```

Comment:
Style question: do we prefer having a whole new class for this, or would it be better to always use the `KubernetesCluster` class and just modify `KubernetesCluster.getTreeItem` to set a different `contextValue` for Minikube clusters?  I am not sure where it is better to put the responsibility... it is going to be a bit hacky either way...!

Reply:
I feel like contextValue implies new class, but I'm open to other suggestions.

Reply:
Cool, let's stick with what you have.